### ISSUE TITLE
Possible missing feature gate `winuser` for `winapi` needed by eframe

### ISSUE THREAD

**Issue by poly000** [emilk/egui#4049]
```
error[E0432]: unresolved import `winapi::um::winuser`
   --> /home/poly/.cargo/registry/src/rsproxy.cn-0dccff568467c15b/eframe-0.26.1/src/native/app_icon.rs:83:9
    |
83  |     use winapi::um::winuser;
    |         ^^^^^^^^^^^^^^^^^^^ no `winuser` in `um`
    |
note: found an item that was configured out
   --> /home/poly/.cargo/registry/src/rsproxy.cn-0dccff568467c15b/winapi-0.3.9/src/um/mod.rs:290:37
    |
290 | #[cfg(feature = "winuser")] pub mod winuser;
    |                                     ^^^^^^^
    = note: the item is gated behind the `winuser` feature
```

Using workaround:

```toml
winapi = { version = "0.3.9", features = ["winuser"] }
```

**Comment by poly000** [comment]
Strange. I've noticed there is

https://github.com/emilk/egui/blob/3672b150ed2c63f87f2a2c8c86ba639f0bbc7d81/crates/eframe/Cargo.toml#L185-L186

But when I cross compile a `x86_64-pc-windows-gnu` binary, cargo complains for missing module.

**Comment by emilk** [comment]
Fix in 0.26.2: https://github.com/emilk/egui/releases/tag/0.26.2
